app/Domains/Accounts/Actions/UpdateAccount.php

```diff
@@ -33,7 +33,11 @@ public function handle(Account $account, array $params): Account
     public function rules(): array
     {
         return [
-
+            'id' => ['required', 'exists:accounts,id'],
+            'name' => ['sometimes', 'string', 'max:255'],
+            'balance' => ['sometimes', 'integer', 'min:0'],
+            'type' => ['sometimes', 'in:' . implode(',', [AccountTypesEnum::CASH, AccountTypesEnum::BANK, AccountTypesEnum::MOBILE])],
```

Comment:
1. AccountTypesEnum has to be injected into the file before it can used.
2. Also, use `AccountTypesEnum::asArray()` instead, this would be a more dynamic approach.
If we make changes to `AccountTypesEnum` in the future, we'll also have to change this file with this approach.